### DISCUSSION TITLE
SerialNumber

### DISCUSSION THREAD

**Discussion by NameNotAvailiable** [svenerbe/ecoflow_dynamic_power_adjustment#6] [Q&A] [answered]
Moin,

im script ist kommentiert, dass die beiden keys und die "serialnumber" des Gerätes einzugeben ist. Soweit so gut.

Mir ist nicht ganz klar, ob die einmalige Eingabe der Seriennummer nach der Kommentierung ausreicht (Zeile 122), oder jedesmal die Seriennummer zu setzen ist (Zeilen 124, 131, 160 usw.), wenn der Begriff im script zu finden ist.

Vielen Dank für Eure Hilfe
Franke

PS: pyscrpiting ist nicht so meins ...

**Comment by svenerbe** [comment, marked as answer]
Bitte readme lesen. Da steht nix von serialnumber ersetzen. Das Script ist für Home Assistent. Die serial number wird beim Aufruf des Service im Home Assistant übergeben. Danke.
